ddl/column.go

```diff
@@ -276,6 +300,16 @@ func setColumnsState(columnInfos []*model.ColumnInfo, state model.SchemaState) {
 	}
 }
 
+func checkPosition(tblInfo *model.TableInfo, pos *ast.ColumnPosition) error {
+	if pos != nil && pos.Tp == ast.ColumnPositionAfter {
```

Comment:
What if the tp is not ColumnPositionAfter?

Reply:
If the tp is not `ColumnPositionAfter`, it should be `ColumnPositionFirst` or `ColumnPositionNone`. In both cases, it should not report an error.